### ISSUE TITLE
node 1.0.4 x86 package not found

### ISSUE THREAD

**Issue by matthewp** [appveyor/ci#500]
Is iojs not supported any more?  Very simple config:

```
# Test against this version of Node.js
environment:
  matrix:
  # node.js
  - nodejs_version: "0.10"
  - nodejs_version: "0.12"
  # io.js
  - nodejs_version: "1.0"

# Install scripts. (runs after repo cloning)
install:
  # Get the latest stable version of Node.js or io.js
  - ps: Install-Product node $env:nodejs_version
  # install modules
  - npm install

# Post-install test scripts.
test_script:
  # Output useful info for debugging.
  - node --version
  - npm --version
  # run tests
  - npm test

# Don't actually build.
build: off
```

**Comment by FeodorFitsner** [comment]
Why would you need io.js 1.0.x?

**Comment by matthewp** [comment]
Does it matter why? The guide says it's supported and even shows up how to set it up.  Is AppVeyor dropping support for io.js with no notice?

**Comment by FeodorFitsner** [comment]
Was just curious. We are not dropping its support without notice. It's just new image re-installed from scratch and some versions were omitted with assumption they are no longer needed. Will add them back.

**Comment by matthewp** [comment]
Ok, great, the reason I'm supporting 1.0.x currently is that several of my projects are not compatible with npm 3 which is the default with Node 4 and 5. Once npm 3 compatibility is fixed we'll likely use Node 4.0.x in CI as that's the LTS release.

**Comment by FeodorFitsner** [comment]
Got it, thanks!
